scripts/dockerfiles/Dockerfile.metrics

```diff
@@ -2,7 +2,6 @@ ARG golang_image
 
 FROM $golang_image as builder
 WORKDIR /go/src/github.com/aws/amazon-vpc-cni-k8s
-ARG GOARCH
```

Comment:
i think we should make this metrics-helper a multi-arch image as well to be consistent.

Reply:
ack, will check and make changes to metrics-helper if needed in a followup PR.